inst/include/child_model_simulation.hpp

```diff
@@ -317,30 +325,30 @@ void adjust_optAB_transmission_rate(int t,
                                     const State<ModelVariant, real_type> &state_curr,
                                     State<ModelVariant, real_type> &state_next,
                                     IntermediateData<ModelVariant, real_type> &intermediate) {
-  const auto cpars = pars.children.children;
   static_assert(ModelVariant::run_child_model,
                 "adjust_optAB_transmission_rate can only be called for model variants where run_child_model is true");
+  const auto& p_hc = pars.children.children;
+  auto& i_hc = intermediate.children;
 
-  //Option A and B were only authorized for women with greater than 350 CD4, so if the percentage of women
-  //on option A/B > the proportion of women in this cd4 category, we assume that some must have a cd4 less than 350
-  //option AB will be less effective for these women so we adjust for that
+  // Option A and B were only authorized for women with greater than 350 CD4, so if the percentage of women
+  // on option A/B > the proportion of women in this cd4 category, we assume that some must have a cd4 less than 350
+  // option AB will be less effective for these women so we adjust for that
   internal::calc_wlhiv_cd4_proportion(t, pars, state_curr, state_next, intermediate);
 
-  if ((intermediate.children.PMTCT_coverage(0) + intermediate.children.PMTCT_coverage(1)) > intermediate.children.prop_wlhiv_gte350) {
-    if (intermediate.children.prop_wlhiv_gte350 > 0) {
-      intermediate.children.excessratio = ((intermediate.children.PMTCT_coverage(0) + intermediate.children.PMTCT_coverage(1)) / intermediate.children.prop_wlhiv_gte350) - 1;
+  auto total_PMTCT_coverage = i_hc.PMTCT_coverage(0) + i_hc.PMTCT_coverage(1);
```

Comment:
common variable extracted

Reply:
change name from total_PMTCT_coverage to optionAB_coverage

Reply:
or option_A_B_coverage if that is preferred